cpp/include/cudf/types.h

```diff
@@ -156,6 +186,29 @@ typedef enum {
   GDF_NUM_COLORS, /** Add new colors above this line */
 } gdf_color;
 
+
+/**---------------------------------------------------------------------------*
+ * @brief Types of binary operations that can be performed on data.
+ * 
+ *---------------------------------------------------------------------------**/
+typedef enum {
+  GDF_ADD,            /**< operator + */
+  GDF_SUB,            /**< operator - */
+  GDF_MUL,            /**< operator * */
+  GDF_DIV,            /**< operator / using common type of lhs and rhs */
+  GDF_TRUE_DIV,       /**< operator / after promoting type to floating point*/
```

Comment:
How is this "true"? That's rather subjective... consider a name change. Perhaps `GDF_FRACTIONAL_DIVISION`? Something involving the words "floating point"?

Reply:
This is named after the [python operator](https://docs.python.org/3/library/operator.html#operator.truediv)